src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +99,59 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllAndSpecificFieldTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        String fieldName = "age";
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT * as a, %s as t " +
```

Comment:
What is `* as a` supposed to do? I am unfamiliar with this syntax.

Reply:
To give an alias to all the field names involved, trying to replicate the same conditions as the original issue

Reply:
Update- you're right, the syntax is invalid as I tested against the incomplete _opendistro/_sql plugin. Will make change for next commit